hacs.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Dreo Smart Device Integration",
+    "name": "Dreo Smart Device Integration with Heaters",
```

Comment:
Remove.  Heaters are Smart Devices.

Reply:
That shouldn't have been in the diffs - I had done it just to differentiate my fork in a list. It's not going to go into the merge.

Reply:
Removed.